src/components/tipRecords/TipRecordComponent.vue

```diff
@@ -31,7 +31,9 @@
         </div>
       </div>
     </div>
-    <div class="tip__article col-4" v-if="isPreviewToBeVisualized(tip)">
+    <div class="tip__article col-4 position-relative" v-if="isPreviewToBeVisualized(tip)">
```

Comment:
`isPreviewToBeVisualized` also within this?

Reply:
These are minor changes depending on one another. That's why I combined it.

Reply:
but the if isPreviewToBeVisualized is also nested inside which doesn't make sense to me

Reply:
I caught your point, the section should be hidden if there are no results. Only one check will stay